cvat-ui/src/components/tasks-page/top-bar.tsx

```diff
@@ -25,13 +28,71 @@ export default function TopBarComponent(props: VisibleTopBarProps): JSX.Element
             <Row justify='center' align='middle'>
                 <Col md={11} lg={9} xl={8} xxl={7}>
                     <Text className='cvat-title'>Tasks</Text>
-                    <Input.Search
-                        className='cvat-task-page-search-task'
-                        defaultValue={searchValue}
-                        onSearch={onSearch}
-                        size='large'
-                        placeholder='Search'
-                    />
+                    <CVATTooltip
```

Comment:
Can you add this tooltip to project search field too?

Reply:
Sure

Reply:
Reworked